rosidl_generator_cpp/resource/idl__type_support.hpp.em

```diff
@@ -0,0 +1,78 @@
+// generated from rosidl_generator_cpp/resource/idl__type_support.hpp.em
+// with input from @(package_name):@(interface_path)
+// generated code does not contain a copyright notice
+@
+@#######################################################################
+@# EmPy template for generating <idl>__struct.h files
+@#
+@# Context:
+@#  - package_name (string)
+@#  - interface_path (Path relative to the directory named after the package)
+@#  - content (IdlContent, list of elements, e.g. Messages or Services)
+@#######################################################################
+@{
+from rosidl_cmake import convert_camel_case_to_lower_case_underscore
```

Comment:
This has now been deprecated, so this should be:

```suggestion
from rosidl_pycommon import convert_camel_case_to_lower_case_underscore
```

Reply:
bb549dc